lib/foreman/cast.rb

```diff
@@ -0,0 +1,22 @@
+module Foreman
+  module Cast
+    def self.to_bool(value)
+      case value
+
+        when String
```

Comment:
@daviddavis see [this](https://github.com/theforeman/foreman/blob/develop/.rubocop_todo.yml#L152-L153).